tasks_downstream.py

```diff
@@ -46,9 +46,14 @@
         "build"
     ]["args"]["ODOO_VERSION"]
 )
-DOCKER_COMPOSE_CMD = f"{shutil.which('docker')} compose" or shutil.which(
-    "docker-compose"
+
+docker_compose_result = subprocess.run(
+    [which("docker"), "compose"], capture_output=True, text=True
```

Comment:
This parameter isn't needed really
```suggestion
    [which("docker"), "compose"], capture_output=True
```